packages/eslint-plugin/docs/rules/return-await.md

```diff
@@ -1,4 +1,4 @@
-# Require/Disallow returning awaited values in specific contexts (return-await)
+# Enforces consistent returning of awaited values in specific contexts (`return-await`)
```

Comment:
this sentence sound weird

Reply:
i wouldn't worry about "in specific contexts", I think it describes well enough without that.